tla-bmcmt/src/main/scala/at/forsyte/apalache/tla/bmcmt/rules/aux/CherryPick.scala

```diff
@@ -629,22 +677,26 @@ class CherryPick(rewriter: SymbStateRewriter) {
 
     val maxLen = elemsOfMemberSets.map(_.size).reduce((i, j) => if (i > j) i else j)
     assert(maxLen != 0)
-    val maxPadded = elemsOfMemberSets.find(_.size == maxLen).get // existence is guaranteed by maxLen
+    // existence is guaranteed by maxLen, but since we use it to pad, we set all pointers to False
```

Comment:
What is "it"?

Reply:
`maxPadded`; it gets used a few lines below, as a stand-in for a statically empty set, to zip with.

Reply:
Is the following accurate? If so, I'd find it clearer

```suggestion
    // existence is guaranteed by maxLen, but we use maxPadded below to pad, so set all pointers to False
```